tests/TODO.md

```diff
@@ -15,6 +15,45 @@ functional unit or feature. The format of list is as follows:
 
 ### TESTS
 
+#### Backend:
+
+* **Stat commit errors**. This test checks if backend's `stat_commit_errors` is
+  taken into account during `Backend` status recalculation (when errors occur,
+  backend must be turned read-only).
+  1. *First error*. This test case checks basic functionality. Backend is first
+  initialized and enabled, then the status is calculated. It must be `OK`.
+  The next `BackendStat` is coming with non-zero `stat_commit_errors`. After the
+  status recalculation it must become `RO`.
+  2. *Subsequent errors*. This case checks if backend remains in state `RO`
+  both when nothing changes and when new errors occur. Send two updates:
+  stat with the same number of commit errors and update with greater number
+  of errors.
+  3. *Restart*. This case checks if `Backend` turns into state `OK` after
```

Comment:
1) If it became read-only due to stat_commit_errors it will be `OK` after restart.
2) In this case it will be `OK` after restart. We don't know when an error occurred so it will be clear only after the next statistics update.